exp-player/addon/components/exp-video-config.js

```diff
@@ -0,0 +1,46 @@
+import ExpFrameBaseComponent from 'exp-player/components/exp-frame-base';
+import layout from 'exp-player/templates/components/exp-video-config';
+
+export default ExpFrameBaseComponent.extend({
+    layout,
+    videoRecorder: Em.inject.service(),
```

Comment:
Ah, so the jshint error can be suppressed then.

`exp-player/addon/components/exp-video-config.js: line 6, col 20, 'Em' is not defined.`

(see "ignore" on this page: http://jshint.com/docs/ )